lib/game.rb

```diff
@@ -12,27 +13,37 @@ def initialize(player_one, player_two, display, messages)
     @messages = messages
   end
 
-  def run
+  def play
     welcome_users
-    @player_one.choose
-    @player_two.choose
-    determine_winner
-    present_winner
+    @player_one.get_input
+    @player_two.get_input
+    result = option_one_result(@player_one.player_input, @player_two.player_input)
+    present_winner(result)
   end
 
   def welcome_users
     @display.present(@messages.welcome_message)
   end
 
-  def determine_winner
-    rules = Rules.new(@player_one.user_input, @player_two.user_input)
-    @winning_option = rules.outcome
+  def option_one_result(option_one, option_two)
+    turn = [option_one, option_two]
+    if option_one == option_two
+      "draw"
+    elsif turn.include?("rock") and turn.include?("scissors") and option_one == "rock"
```

Comment:
You're checking at the end of this statement that Option One must be 'rock' - then you're asking that a 'rock' and a 'scissors' are included in the two-item array, which is the only situation in which this statement returns true... can you see duplication/ problems with that?